health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/TableSearchBar.js

```
@@ -0,0 +1,62 @@
+        value={query}
+        onChange={handleInputChange}
+        style={styles.input}
+      />
+      <div style={styles.iconContainer}>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 24 24"
+          width="16"
+          height="16"
+          fill="#888"
+        >
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)